airflow/auth/managers/simple/simple_auth_manager.py

```diff
@@ -122,7 +124,12 @@ def get_url_logout(self) -> str:
         return url_for("SimpleAuthManagerAuthenticationViews.logout")
 
     def get_user(self) -> SimpleAuthManagerUser | None:
-        return session["user"] if self.is_logged_in() else None
+        if not self.is_logged_in():
+            return None
+        if self.appbuilder.get_app.config.get("SIMPLE_AUTH_MANAGER_ALL_ADMINS", False):
+            return SimpleAuthManagerUser(username="Anonymous", role="admin")
```

Comment:
Small nit:
I assume that the username is not case sensitive - but in the code level we might be consistent (i.e., `Anonymous` or `anonymous`) to prevent potential security bugs. Example from the codebase:

https://github.com/apache/airflow/blob/c051d0aa4c7215b35a4390822aa6ece3ac945900/airflow/www/decorators.py#L91

Reply:
Fair enough